<?xml version="1.0"?>
<launch>
  <arg name="gps_origin_frame" default="gps_origin" />
  <arg name="gps_base_frame" default="gps_base_link" />
  <arg name="map_frame" default="map" />
  <arg name="odom_frame" default="odom" />
  <arg name="base_frame" default="base_link" />
  <arg name="save_on_exit" default="true" />

  <node pkg="map_origin_broadcaster" type="map_origin_broadcaster_node" name="map_origin_broadcaster" output="screen" >
    <param name="gps_origin_frame" value="$(arg gps_origin_frame)" />
    <param name="gps_base_frame" value="$(arg gps_base_frame)" />
    <param name="map_frame" value="$(arg map_frame)" />
    <param name="odom_frame" value="$(arg odom_frame)" />
    <param name="base_frame" value="$(arg base_frame)" />
    <param name="save_on_exit" value="$(arg save_on_exit)" />
  </node>
</launch>
